src/Microsoft.IdentityModel.Protocol.Extensions/project.json

```diff
@@ -3,8 +3,8 @@
     "description": "Provides types that provide support for OpenIdConnect protocol.",
     "dependencies": {
         "Newtonsoft.Json": "6.0.6",
-        "System.IdentityModel.Tokens": "5.0.0-beta4-*",
-        "Microsoft.IdentityModel.Logging":  "1.0.0-beta4-*"
```

Comment:
Having beta4 was causing the warnings regarding the versions when building the project. The fix was either to change the     "version": "2.0.0-*" to "version: 2.0.0-beta-*" in project.json or remove the "beta" from the dependencies. This is what asp.net does too. The beta is added in the build scripts.